README.md

```diff
@@ -9,35 +9,45 @@
 
 # SignalFlow library for JavaScript
 
-## Installation
+## Install the client
 
 To install using npm:
 
 ```sh
-$ npm install signalflow
+npm install signalflow
 ```
 
 ### Supported Node.js versions
 
 | Version | Node.js         |
 | ------- | --------------- |
-| `8.x.x` | `>=12.10.0 <=20` |
-| `7.4.x` | `>=8.0.0 <18`   |
-| `7.3.1` | `>=8.0.0 <11`   |
+| `8.x.x` | `>=12.10.0 <= 20` |
+| `7.4.x` | `>=8.0.0 < 18`   |
+| `7.3.1` | `>=8.0.0 < 11`   |
 
 ## Usage
 
-### Configuring the Signalflow websocket endpoint
+### Configure the SignalFlow websocket endpoint
 
-If the websocket endpoint is not set manually, this library uses the `us0` realm by default.
-If you are not in this realm, you will need to explicitly set the
-endpoint urls above. To determine if you are in a different realm and need to
-explicitly set the endpoints, check your profile page in the SignalFx
-web application.
+The default realm is ``us0``, unless you manually set the websocket endpoint.
+If you aren't in the ``us0`` realm, you must explicitly set the following
+environment variables:
 
-### Examples
+| Environment variable                          | Endpoint                                        |
+|-----------------------------------------------|-------------------------------------------------|
+| `DEFAULT_INGEST_ENDPOINT`                     | `https://ingest.<realm>.signalfx.com`           |
+| `DEFAULT_API_ENDPOINT`                        | `https://api.<realm>.signalfx.com`              |
+| `DEFAULT_SIGNALFLOW_WEBSOCKET_ENDPOINT`       | `wss://stream.<realm>.signalfx.com`             |
```

Comment:
@seemk Does this work?

Reply:
These aren't environment variables (at least I didn't find them), I think this part can be omitted.